src/ctnetlink/nlas/flow/nla.rs

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+
+use byteorder::{BigEndian, ByteOrder};
+use netlink_packet_utils::{
+    nla::{DefaultNla, Nla, NlaBuffer, NLA_F_NESTED},
+    Emitable, Parseable,
+};
+
+use crate::constants::{
+    CTA_ID, CTA_MARK, CTA_PROTOINFO, CTA_STATUS, CTA_TIMEOUT, CTA_TUPLE_ORIG,
+    CTA_TUPLE_REPLY, CTA_USE,
+};
+
+use super::{
+    ip_tuple::TupleNla, protocol_info::ProtocolInfo, status::ConnectionStatus,
+};
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum FlowNla {
```

Comment:
How about `FlowAttribute`?

Nla is not self-describing.

Reply:
Ok,
Should I also rename `StatNla` and `TupleNla`?